packages/focuslock/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
why create a whole new package for this?  It's only used in the Dialog.  Why not colocate?

Reply:
i have plans for reuse